### PR TITLE
`General`: Add missing translations to feedback groups

### LINKED ISSUE #6390: `Feedback`: Group headers are not translated
### Describe the bug

The headers of the groups in the new feedback detail view are not translated.

### To Reproduce

1. Switch Artemis to German.
2. Open the feedback details for a result.

### Expected behavior

The group names are translated.

### Screenshots

Screenshot 2023-03-28 at 11-25-31 Programmieraufgaben

### Which version of Artemis are you seeing the problem on?

6.1.1

### What browsers are you seeing the problem on?

Firefox

### Additional context

https://github.com/ls1intum/Artemis/blob/c14226f0417f505003000ff0153284edce3b9c1d/src/main/webapp/app/exercises/shared/feedback/node/feedback-node.component.html#L22

### Relevant log output

_No response_

### PR DESCRIPTION
**Pull request by simon-hng**
#### General

- [x] This is a small issue that I tested locally and was confirmed by another developer on a test server.

### Motivation and Context

Closes #6390

### Description

Adds translations strings for following groups and uses them:

- wrong
- warning
- info
- correct

### Steps for Testing

Prerequisites:
- Exercise with feedback

1. Go to an exercise with feedback 
2. Switch language to German and see if it is correctly translated

### CODE REVIEW

**Review by b-fein — DISMISSED** [review]
Thanks, looks good.

**Review by Strohgelaender — APPROVED** [review]

**Review by terlan98 — DISMISSED** [review]
Tested on TS2. Feedback headers are now translated.

File: src/main/webapp/app/exercises/shared/feedback/node/feedback-node.component.html [hunk 1]
```diff
@@ -19,7 +19,9 @@ <h4 class="feedback-item__category">{{ feedbackItem.name }} {{ feedbackItem.titl
     <div class="feedback-group" [ngClass]="'bg-' + feedbackItemGroup.color" (click)="feedbackItemGroup.open = !feedbackItemGroup.open">
         <div class="feedback-group__left">
             <fa-icon class="me-2" [icon]="feedbackItemGroup.open ? faAngleUp : faAngleDown"></fa-icon>
-            <h4 class="feedback-item__category">{{ feedbackItemGroup.name | titlecase }} ({{ feedbackItemGroup.members.length }})</h4>
+            <h4 class="feedback-item__category">
+                {{ 'artemisApp.feedback.group.' + feedbackItemGroup.name | artemisTranslate | titlecase }} ({{ feedbackItemGroup.members.length }})
```
Comment (JohannesStoehr) [inline src/main/webapp/app/exercises/shared/feedback/node/feedback-node.component.html:23]: Why do we need titlecase here, when we can already capitalize the translations in the file?

Comment (simon-hng) [reply src/main/webapp/app/exercises/shared/feedback/node/feedback-node.component.html:23]: Just in case it won't be capitalized some time

Comment (simon-hng) [reply src/main/webapp/app/exercises/shared/feedback/node/feedback-node.component.html:23]:
```suggestion
                {{ 'artemisApp.feedback.group.' + feedbackItemGroup.name | artemisTranslate }} ({{ feedbackItemGroup.members.length }})
```

**Review by JohannesStoehr — DISMISSED** [review]
Code changes look good to me. Thanks for refactoring the titleCase pipe.

**Review by bassner — DISMISSED** [review]
LGTM 👍

**Review by terlan98 — APPROVED** [review]
Reapproved. The changes look good.

**Review by JohannesStoehr — APPROVED** [review]

**Review by b-fein — APPROVED** [review]

**Review by Strohgelaender — APPROVED** [review]

**Pull request merged**